spec/integration/iso_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'ISO' do
```

Comment:
Same as with DC. This looks like it could be handled by the default `find_by_id_and_name` specs